pre-check/v1.x/check.sh

```diff
@@ -592,6 +592,17 @@ check_backup_target()
 check_images()
 {
     log_info "Starting Longhorn Backing Images check..."
+
+    version=$(kubectl get settings.harvesterhci.io server-version -o json | jq -r '.value')
+
+    # If version is before v1.4.x, print a message and return
+    if [[ $version =~ ^v([0-9]|1\.[0-3])\..* ]]; then
```

Comment:
```suggestion
    if [[ $version =~ ^v([0-1]|\.[0-3])\..* ]]; then
```
Or check one major, since this script is specific for v1.x. 
```suggestion
    if [[ $version =~ ^v(1\.[0-3])\..* ]]; then
```